frontend/src/views/pages/dealerships/DealershipAdd.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div>
+    <CAlert show color="danger" v-if="errorMessage" class="mb-2">{{ errorMessage }}</CAlert>
+    <CForm @submit.prevent="submit">
+      <CInput
+        label="Dealership Name"
+        :lazy="false"
+        :value.sync="form.name"
+        placeholder="Dealership Name"
+      />
+
+      <CTextarea
+        label="Dealership Description"
+        :lazy="false"
+        :value.sync="form.description"
+        placeholder="Dealership Description"
+        rows="6"
+      />
+      <CRow class="justify-content-center">
+        <CButton 
+          color="primary"
+          type="submit"
+          id = "create-dealership"
+          :disabled="disableButtons"
+        >
+          Create
+        </CButton>
+        <CButton 
+          class="ml-1"
+          color="danger"
+          :disabled="disableButtons"
+        >
+          Cancel
+        </CButton>
+      </CRow>
+    </CForm>
+  </div>
+</template>
+
+<script>
+const axios = require('axios');
+
+export default {
+  name: 'DealershipAdd',
+  data() {
+    return {
+      form: this.getEmptyForm(),
+      submitted: false,
+      errorMessage: null,
+      disableButtons: false
+    }
+  },
+  methods: {
+    checkIfValid () {
+      return !!this.form.name && this.form.name != '';
+    },
+    submit () {
+      this.disableButtons = true;
+      if (this.checkIfValid()) {
+        // post request to API to create the new dealership
+        axios({
+          method: 'POST',
+          url: `${this.$store.state.api}/dealerships`,
+          headers: {
+            'Authorization': `Bearer ${this.$store.state.auth.token}`
+          },
+          data: {
+            name: this.form.name,
+            description: this.form.description,
+            admin: this.form.admin
+          }
+        }).then(response => {
+          if (response.data.success) {
+            // if (!this.$store.state.auth.createDealershipCompleted) {
+            //   this.$store.commit('setProperty', 'createDealershipCompleted', true);
+            // }
+            this.$router.go();
+          } else {
+            console.log(response);
+            this.showErrorMessage("Unable to create the new dealership.")
+            this.disableButtons = false;
+          }
+        }).catch(err => {
+          console.log(err);
+          this.showErrorMessage("Unable to create the new dealership.")
+          this.disableButtons = false;
+        })
+      } else {
+        this.showErrorMessage('Must have a proper dealership name.');
+        this.disableButtons = false;
+      }
+    },
+    getEmptyForm () {
+      return {
+        name: "",
+        description: "",
+        admin: this.$store.state.auth.userId
+      }
+    },
+    showErrorMessage(message) {
+      this.errorMessage = message;
+      setTimeout(() => {
+        this.errorMessage = null;
+      }, 5000)
+    }
+  }
+}
+</script>
+
+<style>
+
+</style>
```

Comment:
Empty tags

Reply:
Fixed in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/37/commits/8c83bea65e9c8a6186a069f6415340392a55a7b4)